### ISSUE TITLE
Ecosystem: flag which version of Thymeleaf is supported

### ISSUE THREAD

**Issue by maxxyme** [thymeleaf/thymeleaf.github.com#25]
Hello,

As a new user of Thymeleaf, I started with the latest version, 3.0.2.RELEASE
I also found some great dialects under http://www.thymeleaf.org/ecosystem.html

Unfortunately, some don't seem to be actively supported, and are not compatible with Thymeleaf 3.
e.g. HTML5 Validation Dialect
see http://stackoverflow.com/questions/40870051/html5-related-dialect-compatible-with-thymeleaf-3

So, could you please update the Ecosystem page by flagging which version of Thymeleaf each element supports?

**Comment by ultraq** [comment]
Hey @danielfernandez, I want to give the ecosystem page some updating and so I can come up with some way of tagging projects as Thymeleaf 2/3 compatible and all that, but do you think you could also provide a list of ecosystem projects that can be added/removed as well?  There are a lot of new projects out there since we last gave this a look for the website redesign (like all the ones being retweeted by the Twitter account) and existing projects that don't seem to be getting much love/attention from their maintainers any more.

**Comment by danielfernandez** [comment]
Yes I can. But give me a couple of days please :). We should think about what to do about ecosystem projects not updated to Thymeleaf 3, which is 7 months old already. I would definitely keep the *official* ones somehow (conditional comments, tiles2) but lower their relevance *a lot*... but to be honest, I'm not sure if we should keep the non-official ones that are not updated... things like Dandelion or so... what do you think?
